conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,202 @@
+and fair.
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee"), currently comprised of [*note:* still gathering volunteers, this will be filled in as they become available and prior to final upload into the rendered site]:
+
+* Person One foo@email (Chair of the Committee)
+* ...
+* Person Five bar@email
+
+The Jupyter Steering Council will approve the members of this committee,
+comprised of five members. The members will be drawn from our entire community,
+and we intend that it *not* be only Steering Council members: anyone who is
+actively engaged with Jupyter can propose to join this committee (we'll
+explicitly solicit volunteers when short of five members).  One member will be
+designated chair of the group and will be responsible for all reports back to
+the Steering Council.
+
+
+## Enforcement guidelines and principles
+
+Enforcing the Code of Conduct impacts our community today and for the future.
+It's an action that we do not take lightly. When reviewing enforcement
+measures, the Code of Conduct Committee will keep the following values and
+guidelines in mind:
+
+* Act in a personal manner rather than impersonal.  The Committee can engage
+  the parties to understand the situation, while respecting the privacy and any
+  necessary confidentiality of reporters.  But sometimes it is necessary to communicate with one or more individuals directly: the Committee's goal is to improve the health of our community rather than only produce a formal decision.
+
+* Emphasize empathy for individuals rather than judging behavior, avoiding
+  binary labels of "good" and "bad/evil". Overt, clear-cut agression and
+  harrassment exists and will be addressed unambiguously.  But many scenarios
+  that can prove challenging to resolve are those where normal disagreements
+  devolve into inappropriate behavior from multiple parties.  Understanding the
+  full context and finding a path that re-engages all is hard, but ultimately
+  the most productive for our community.
+
+* Increase engagement in good discussion practice.
+
+* Be mindful of the needs of new members: provide them with explicit support
+  and consideration, with the aim of increasing participation from
+  underrepresented groups in particular.
+
+* Individuals come from different cultural backgrounds and native languages.
+  While lack of intent to harm is not an excuse, try to identify an honest
+  misunderstanding caused by a non-native speaker and help them understand the
+  issue and how to change.  Complex discussion in a foreign language can be very intimidating, and we want to grow our diversity also across nationalities and cultures.
+
+* Our actions will reflect compassion for all individuals. We will seek to
+  understand, to educate, and, as necessary, take action.
+
+
+**Mediation:** voluntary, informal mediation is a tool at our disposal.  In
```

Comment:
Thanks @ellisonbg. Good points re: mediation and mutual prior conflict. I also see mediation as a reasonable option when both parties agree and there has simply been miscommunication or misunderstanding.

My experience with CoC violations, as a member of a CoC committee and as a safe advocate for women in tech, has been to deal with cases that are far beyond verbal conflict that include physical harassment, sexual aggressiveness especially when alcohol is mixed in, stalking, doxxing, etc. These sadly do happen in open source. When dealing with cases such as these, mediation is usually not a viable option as personal safety concerns are foremost.